From a48cdf040a9f7c208bcb37c904d8635d9b61e076 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <karel.koci@nic.cz>
Date: Wed, 5 Jun 2019 13:01:43 +0200
Subject: [PATCH] fstools: block-mount: fix restart for fstab

Default restart calls stop and then start. That causes this service to
just call block unmount but they are never mounted back.
---
 package/system/fstools/Makefile         | 2 +-
 package/system/fstools/files/fstab.init | 4 ++++
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/package/system/fstools/Makefile b/package/system/fstools/Makefile
index f7695af..4302b7a 100644
--- a/package/system/fstools/Makefile
+++ b/package/system/fstools/Makefile
@@ -8,7 +8,7 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=fstools
-PKG_RELEASE:=3
+PKG_RELEASE:=4
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL=$(PROJECT_GIT)/project/fstools.git
diff --git a/package/system/fstools/files/fstab.init b/package/system/fstools/files/fstab.init
index 5faa8ec..afed78e 100644
--- a/package/system/fstools/files/fstab.init
+++ b/package/system/fstools/files/fstab.init
@@ -11,6 +11,10 @@ start() {
 	echo "this file has been obsoleted. please call \"/sbin/block mount\" directly"
 }
 
+restart() {
+	start
+}
+
 stop() {
 	/sbin/block umount
 }
-- 
2.21.0

